src/components/TableChats.vue

```diff
@@ -0,0 +1,167 @@
+        <tr>
+          <th v-for="header in headers" :key="header">{{ header }}</th>
+        </tr>
+      </thead>
+      <tbody class="table-chats__table-body">
+        <tr
+          v-for="(row, rowIndex) in data[activeTab]"
+          :key="rowIndex"
+          class="table-chats__table-row"
+        >
+          <td v-for="(value, colIndex) in row" :key="colIndex">{{ value }}</td>
+        </tr>
+      </tbody>
+    </table>
+    <hr class="table-chats__separator" />
+    <footer class="table-chats__bottom">
+      <TablePagination
+        v-model="chatsCurrentPage"
+        :count="chatsCount"
+        :countPages="chatsCountPages"
+        :limit="chatsLimit"
+      />
+    </footer>
+  </section>
+</template>
+
+<script>
+import TablePagination from '@/components/TablePagination.vue';
+
+export default {
+  name: 'TableChats',
+
+  components: {
+    TablePagination,
+  },
+
+  data: () => ({
+    activeTab: 0,
+    page: 1,
+    tabs: ['Aguardando', 'Em andamento', 'Encerrados'],
+    headers: ['Contato', 'Agente', 'Horário de início', 'Tempo de interação'],
+    data: [
+      [
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+        ['Fernanda da Silva Santos', 'Tamara', '12:59', '01:16'],
+      ],
+      [
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+        ['Monica Cristina', 'Vitória', '12:59', '01:16'],
+      ],
+      [
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+        ['Claudio Leal', 'Larissa', '12:59', '01:16'],
+      ],
+    ],
```

Comment:
Is `data` the most descriptive name you could give?

Reply:
Instead, you could create a mock file that returns this data, to facilitate integration with the backend in the future.